### ISSUE TITLE
Models derivation

### ISSUE THREAD

**Issue by ape364** [tortoise/tortoise-orm#107]
**Describe the bug**
Derived model not derives parent model fields.

**To Reproduce**
[example](https://gist.github.com/ape364/b3f5478853d2a409e3634891e821374d)

Logs:

```
[2019-02-26 16:58:50,616] DEBUG [asyncio.__init__:53] Using selector: EpollSelector
[2019-02-26 16:58:50,648] DEBUG [db_client.create_connection:64] Created connection <asyncpg.connection.Connection object at 0x7f5a6238b2d0> with params: user=postgres database=testdb host=localhost port=5432
[2019-02-26 16:58:50,649] DEBUG [db_client.execute_script:118] CREATE TABLE IF NOT EXISTS "something" ("name" TEXT NOT NULL, "id" SERIAL NOT NULL PRIMARY KEY);
[2019-02-26 16:58:50,667] DEBUG [db_client.close:76] Closed connection <asyncpg.connection.Connection object at 0x7f5a6238b2d0> with params: user=postgres database=testdb host=localhost port=5432
```

**Expected behavior**
Derived model includes all parent model fields (`created`, `updated`) as described [here](https://tortoise-orm.readthedocs.io/en/latest/models.html#usage).

**Additional context**
Installed versions:
```
Python 3.7.2

tortoise-orm==0.11.4
asyncpg==0.18.3

PostgreSQL 11.2 (Debian 11.2-1.pgdg90+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 6.3.0-18+deb9u1) 6.3.0 20170516, 64-bit
```

**Comment by grigi** [comment]
Hi

Thank you for the detailed issue report :smile: 
Tortoise doesn't currently support derived/subclassed models.

Will use this ticket as a feature request.

**Comment by FirstKlaas** [comment]
I guess this one is solved by version 0.12.4. Please give it a try.
